test/e2e/validation_test.go

```diff
@@ -0,0 +1,96 @@
+// +build e2e
+
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e_test
+
+import (
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	capiv1 "sigs.k8s.io/cluster-api/api/v1alpha3"
+	"sigs.k8s.io/cluster-api/util"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+var _ = Describe("Cluster API Validation", func() {
+	It("verifies validation is setup correctly and works", func() {
```

Comment:
Should this be updated to state that it is attempting validation of `Machine` specifically here? I suspect we should also test validation of the other types as well, otherwise we are only verifying that the `Machine` validation webhooks are functional.

Reply:
Yup. Thanks for pointing this out. I was hoping to add just this test for now and then adding future validation tests for other types.

Reply:
sgtm